src/elements/Table.tsx

```diff
@@ -79,11 +133,16 @@ export function Table<Headers extends string>({
    */
   headers,
 }: Props<Headers>): ReturnType<Component<Props<Headers>>> {
-  const columnWidths = Object.values(headers).map((header) => {
-    if (typeof header !== "string") {
-      return 5;
+  const columns = Object.values(headers).map((header) => {
```

Comment:
I think the code could be simplified by normalizing the headers first and then using the normalized version:

```ts
const normalizedHeaders = Object.keys(headers).reduce((normalizedHeaderAccumulator, key) => ({
  ...normalizedHeaderAccumulator,
  [key]: typeof headers[key] === 'string'
    ? { title: headers[key] }
    : headers[key]
}), {});
```

Reply:
Not 100% sure what you mean, but I've updated my PR. Could you please check on that ? :)

Reply:
Here's the refactor I was talking about:
https://github.com/dbartholomae/jsx-md/blob/main/src/elements/Table.tsx#L133